### PR TITLE
Add some docs on SPDX

### PR DESCRIPTION
**Pull request by isuruf**

### CODE REVIEW

**Comment by isuruf** [comment]
Grayskull deals with only one license and doesn't  generate license expressions, right?

**Comment by jakirkham** [comment]
In any event having multiple licenses is a bit of an unusual use case. Should we not recommend even though it covers many common use cases?

File: src/maintainer/adding_pkgs.rst [hunk 1]
```diff
@@ -663,6 +663,32 @@ In this case, please also notify the upstream developers that the license file i
...
+................................
+
+For the ``about: license`` entry in the recipe ``meta.yaml``, use a SPDX identifier or expression or Other.
+
+See `SPDX license identifiers <https://spdx.org/licenses/>`__ for the licenses.
+See `SPDX license exceptions <https://spdx.org/licenses/exceptions-index.html>`__ for license exceptions.
+See `SPDX specification <https://spdx.org/spdx-specification-21-web-version>`__ Appendix IV
+for the specification on expressions.
+
+.. code-block:: yaml
```
Comment (scopatz) [inline src/maintainer/adding_pkgs.rst]: This doesn't look like yaml 😉

**Comment by jakirkham** [comment]
What if we just start by saying this is recommended?

Admittedly there is some chicken and the egg here, but having a place to point to with a recommendation will be useful. Also it will let us collect early feedback so we can iron out the kinks 😉

**Comment by mbargull** [comment]
> In any event having multiple licenses is a bit of an unusual use case.

Not at all. Upstream projects themselves or our recipes often just do not communicate well enough what the actual licensing terms are, e.g., our [`conda` recipe says `BSD 3-Clause`](https://github.com/conda-forge/conda-feedstock/blob/abced79261284238108e8419fe6d101926acd8b1/recipe/meta.yaml#L121), although the actual license is (as a SPDX license expression) [`BSD-3-Clause AND ISC AND MIT`](https://github.com/conda/conda/blob/4.8.3/LICENSE.txt#L33L37).

As packagers/redistributors we should not care much about the "main" license of a project's own source but look at the actual license terms a distribution/use falls under.

> What if we just start by saying this is recommended (not yet required :slightly_smiling_face:)?

(But I would WANT everyone to use it -- yesterday ;P.)
Yes, I agree we should recommend it at this point.
We could also "strongly recommend" or recommend it with an asterisk like "usage of SPDX expression may become a requirement in the future" ;).

**Comment by marcelotrevisani** [comment]
> Grayskull deals with only one license and doesn't generate license expressions, right?

Yeap, you are right on this point.
I didn't implement the license expressions yet, it is normalizing just one license per time or predicting looking for a license file which license is that.
It should not be difficult to implement or to use the ``license-expression`` package to do this job on grayskull

But I think it would be nice to add ``grayskull`` as a suggestion to generate recipes for python, and it will add the license name following the ``spdx`` as well

**Comment by scopatz** [comment]
I am OK with recomend language, but someone should write a CFEP.

**Comment by isuruf** [comment]
We are now recommending SPDX in the linter although it is not a must (green with a hint).
This documentation should be useful for people looking for guidance. I'll add this URL to the linter after this is merged.

**Comment by ocefpaf** [comment]
@marcelotrevisani once you get some free time do you think it is feasible to use your package to recommend the correct SPDX from in the linter error?

**Comment by marcelotrevisani** [comment]
> @marcelotrevisani once you get some free time do you think it is feasible to use your package to recommend the correct SPDX from in the linter error?

I think it is feasible indeed
Just need to find some free time during this week and I will change the way which ``grayskull`` is getting the ``spdx`` and it will discard the deprecated formats

File: src/maintainer/adding_pkgs.rst (hunk 1 above)
Comment (beckermr) [reply src/maintainer/adding_pkgs.rst]: I fixed this.

**Comment by beckermr** [comment]
xref: https://github.com/conda-forge/cfep/pull/30

**Comment by isuruf** [comment]
Now that the CFEP is in, this is ready.

**Pull request merged**